docs/root/api/client_features.rst

```diff
@@ -0,0 +1,20 @@
+.. _client_features:
+
+Well Known Client Features
+==========================
+
+Authoritative list of features that an xDS client may support. An xDS client supplies the list of
+features it supports in the :ref:`client_features <envoy_api_field_core.Node.client_features>` field.
+Client features use reverse DNS naming scheme, for example `com.acme.feature`.
+
+Currently Defined Client Features
+---------------------------------
+
++------------------------------------------------+-------------------------------------------------+
+| Client Feature Name                            | Description                                     |
++================================================+=================================================+
+| envoy.config.require-any-fields-contain-struct | This feature indicates that xDS client requires |
+|                                                | that the configuration entries of type          |
+|                                                | *google.protobuf.Any* contain messages of type  |
+|                                                | *google.protobuf.Struct* only                   |
```

Comment:
Do you mean `udpa.type.v1.TypedStruct`?

Reply:
I'm not sure. This is for gRPC and came from @markdroth

Reply:
Yeah, we want `TypeStruct`. gRPC will be able to consume this, since it's only a simple wrapper around `Struct`.

Reply:
done